### PR TITLE
Standalone builds for all OS

### PR DESCRIPTION
**Pull request by SubJunk**

### CODE REVIEW

**Comment by SubJunk** [comment]
@valib I think this is working for all operating systems now - I have tested on Windows and macOS, and it just needs testing on Linux. I am thinking of releasing it as UMS 9.0.0-b1 to get testers.

**Comment by valib** [comment]
@SubJunk it is not wrking for me because here https://github.com/UniversalMediaServer/UniversalMediaServer/blob/598c43e127b4e612fd713901327f1da1e53a583b/pom.xml#L1162 are missing definitions to make dir and subdirs for `jre-x64` and `jre-x86`

**Comment by SubJunk** [comment]
@valib ah I had manually copied the JRE files over which made me not notice that problem. I'm fixing it now

**Comment by SubJunk** [comment]
@valib should be fixed now

**Comment by VaiTon** [comment]
Has this been merged? @SubJunk

**Comment by SubJunk** [comment]
@VaiTon not yet, I will release a public beta before merging, to make sure it works well for everyone

**Comment by SubJunk** [comment]
Ok I have been doing some testing and it's working well for me so will release the beta soon unless you object @valib

File: pom.xml [hunk 1]
```diff
@@ -1091,12 +1091,6 @@
 		-->
 		<profile>
 			<id>windows</id>
-			<activation>
-				<activeByDefault>false</activeByDefault>
-				<os>
-					<family>windows</family>
-				</os>
-			</activation>
```
Comment (valib) [inline pom.xml]: @SubJunk this deletion stopped making the "UMS-setup.exe". Do not ask me why. I realized it when I deleted the Eclipse "target" folder and tried to make distributable files.

Comment (SubJunk) [reply pom.xml]: Which command do you use? I do `mvn -P windows package`

**Comment by valib** [comment]
I make it in Eclipse and use the standard setting for build from Wiki https://github.com/UniversalMediaServer/UniversalMediaServer/wiki/Development

**Comment by SubJunk** [comment]
@valib before this change, I was unable to build only for Linux via Windows - if I ran `mvn -P linux package` it would build for both Windows and Linux, so that was making the linux builds take twice as long to build. Maybe there's a way to get Eclipse to specify a profile?

**Pull request merged**